### PR TITLE
Prototech 47: Avoid param structs/tuples on external functions.

### PR DESCRIPTION
**Pull request by prat-gpt**
# Description of change
## High level
* See - https://github.com/Fixed-Point-Solutions/prototech-ajna-audit/issues/47
* Update params in `PositionManager` external functions to have distinct parameters rather than struct param

### CODE REVIEW

File: src/PositionManager.sol [hunk 1]
```diff
@@ -85,6 +85,13 @@ contract PositionManager is PermitERC721, IPositionManager, Multicall, Reentranc
         uint256 lpbAmountTo;      // [WAD] the LP awarded in to bucket
     }
 
+    /// @dev Struct used for `memorializePositions` function Lenders Local vars
+    struct LendersBucketLocalVars {
```
Comment (MikeHathaway) [inline src/PositionManager.sol:89]: Can these structs be moved into the interface?

**Review by MikeHathaway — APPROVED** [review]
LGTM - one small comment about moving local structs into interface

File: src/PositionManager.sol (hunk 1 above)
Comment (grandizzy) [reply src/PositionManager.sol:89]: I think is fine and inline with other contracts to stay here as they hold local vars

Comment (MikeHathaway) [reply src/PositionManager.sol:89]: Yeah makes sense if those structs are never used in another context. Happy as is.

**Review by grandizzy — APPROVED** [review]
LGTM

**Pull request merged**